impl/src/test/java/io/kroxylicious/testing/kafka/KafkaClusterTest.java

```diff
@@ -264,8 +374,8 @@ private void produce(KafkaCluster cluster, String topic, String message) throws
                 ProducerConfig.KEY_SERIALIZER_CLASS_CONFIG, StringSerializer.class,
                 ProducerConfig.VALUE_SERIALIZER_CLASS_CONFIG, StringSerializer.class,
                 ProducerConfig.DELIVERY_TIMEOUT_MS_CONFIG, 3_600_000));
-        try (var producer = new KafkaProducer<String, String>(config)) {
-            producer.send(new ProducerRecord<>(topic, "my-key", message)).get();
+        try (var producer = CloseableProducer.wrap(new KafkaProducer<String, String>(config))) {
```

Comment:
Does make me wonder if we should add a ctor to `Closeable*` which accepts `config` and creates the `Kafka*` internally.

Reply:
Yeah, I'll do that.

Reply:
I added create factory methods as that seem most in keeping with the existing API.